internal/topology/check/compatibility.go

```diff
@@ -53,14 +53,14 @@ func ObjectsAreCompatible(current, desired client.Object) field.ErrorList {
 	if currentGK.Group != desiredGK.Group {
 		allErrs = append(allErrs, field.Forbidden(
 			field.NewPath("metadata", "apiVersion"),
-			fmt.Sprintf("group cannot be changed from %v to %v to prevent incompatible changes in %v/%v",
+			fmt.Sprintf("apiVersion.group cannot be changed from %q to %q into %s/%s to prevent incompatible changes in the Cluster",
```

Comment:
> apiVersion.group cannot be changed from "cluster.x-k8s.io/foo" to "cluster.x-k8s.io/bar" into Cluster.cluster.x-k8s.io/foo to prevent incompatible changes in the Cluster

Should this maybe rather be 

> apiVersion.group of Cluster.cluster.x-k8s.io/foo cannot be changed from "cluster.x-k8s.io/foo" to "cluster.x-k8s.io/bar" to prevent incompatible changes in the Cluster

`into` sounds strange to me in this context. But that could absolutely be just me :)

Reply:
fixed